### PR TITLE
V2 small improvements

### PR DESCRIPTION
**Pull request by ciungulete**
- Update php-cs-fixer
- Cleanup code
- Add Money RoundingMode to updatePaymentMethod
- remove code duplicates (create MoneyRoundingMode trait)

### CODE REVIEW

File: .php-cs-fixer.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...st.php":909629863,"tests\/Plan\/ConfigPlanTest.php":1390790695,"tests\/Plan\/ConfigPlanRepositoryTest.php":2731909415,"tests\/Helpers\/HelpersTest.php":21708501,"tests\/EventServiceProviderTest.php":901459568,"tests\/SwapSubscriptionPlanTest.php":3922404503,"tests\/CashierTest.php":1079069368,"tests\/SubscriptionTest.php":494395828,"tests\/BillableTest.php":1455610985,"tests\/BaseTestCase.php":760081172,"tests\/Mollie\/UpdateMolliePaymentTest.php":2278010199,"tests\/Mollie\/BaseMollieInteractionTest.php":4227849375,"tests\/Mollie\/GetMollieMandateTest.php":84204709,"tests\/Mollie\/CreateMollieCustomerTest.php":1350788803,"tests\/Mollie\/GetMollieRefundTest.php":3224566903,"tests\/Mollie\/GetMollieCustomerTest.php":206342844,"tests\/Mollie\/GetMolliePaymentTest.php":2751900370,"tests\/Mollie\/CreateMolliePaymentTest.php":3706303709,"tests\/Mollie\/CreateMollieRefundTest.php":3545454736,"tests\/CashierServiceProviderTest.php":3627601961,"tests\/Credit\/CreditTest.php":3564313542,"tests\/Coupon\/RedeemedCouponTest.php":2494995236,"tests\/Coupon\/CouponOrderItemPreprocessorTest.php":251911886,"tests\/Coupon\/ConfigCouponRepositoryTest.php":3733913546,"routes\/webhooks.php":1754211312}}
```
Comment (sandervanhooft) [inline .php-cs-fixer.cache:1]: Can you remove this file? Add it to .gitignore if necessary

File: src/Traits/MoneyRoundingMode.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Laravel\Cashier\Traits;
+
+use Money\Money;
+
+trait MoneyRoundingMode
```
Comment (sandervanhooft) [inline src/Traits/MoneyRoundingMode.php:7]: Rename to `HandlesMoneyRounding`

**Review by sandervanhooft — CHANGES_REQUESTED** [review]
Some minor stuff, overall looks good.

File: src/Traits/MoneyRoundingMode.php (hunk 2 above)
Comment (ciungulete) [reply src/Traits/MoneyRoundingMode.php]: Done

File: .php-cs-fixer.cache (hunk 1 above)
Comment (ciungulete) [reply .php-cs-fixer.cache]: Done

**Pull request merged**